### ISSUE TITLE
Borders

### ISSUE THREAD

**Issue by capsticm** [tipsy/bubbly-bg#12]
Can I set a border on the bubbles. I want something a little more like a biological cell.
 Wonderful work by the way.

**Comment by tipsy** [comment]
Thanks @capsticm.
Borders are annoying in canvas. You need to set `lineWidth` and `strokeStyle`, then call `stroke()`. This will draw a stroke on the current outline of your shape, so if you have `lineWidth = 4`, you will get 2px inside the shape, and 2px outside. Most people would probably want the stroke to be 4px outside the shape, or be allowed to choose. In addition, drawing the stroke interferes with the default [composite-style](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalCompositeOperation) used in bubbly (lighter), so you would have to change this to make it work "as expected". All of these things would add a lot of complexity (and lines of code), so I thought it wasn't worth it.

### **tl;dr**: No, you can't. I considered adding borders, but it made everything a lot more complicated.

**Comment by capsticm** [comment]
G'day David
Thank you for the reply and information regarding borders.
Might try and have a fiddle with it myself.
Thanks again.
BTW I reckon there would be a lot of interest if you created a tutorial
showing how you built this.
regards
mikec
